docs/quickstart.rst

```diff
@@ -54,15 +54,15 @@ For example, there are several visualizers that act as transformers, used to per
 
     visualizer = ParallelCoordinates()
     visualizer.fit_transform(X, y)
-    visualizer.poof()
+    visualizer.poshowof()
```

Comment:
Was just curious and clicked through to the diff and quickly scrolling through. Looks like a mistake here.

Reply:
Good catch!